gapis/service/service.proto

```diff
@@ -1078,8 +1078,8 @@ message ProfilingData {
     message Group {
       int32 id = 1;
       string name = 2;
-      Group parent = 3;
-      path.Command link = 4;
+      int32 parent = 3; // references Group.id
```

Comment:
Instead of commenting should we name it to parentID?

Reply:
Done.